src/common/serviceUtils.js

```diff
@@ -87,25 +86,32 @@ function handleApiMessages(messages) {
 export function handleRequestError(error) {
   // Validation errors
   if (error.response && error.response.data.field_errors) {
-    logInfo('Field Errors', error.response.data.field_errors);
+    logInfo('API field errors returned to client in field_errors field', { fieldErrors: error.response.data.field_errors });
     handleFieldErrors(error.response.data.field_errors);
   }
 
   // API errors
   if (error.response && error.response.data.errors !== undefined) {
-    logInfo('API Errors', error.response.data.errors);
+    logInfo('API errors returned to client in errors field', { errors: error.response.data.errors });
     handleApiErrors(error.response.data.errors);
   }
 
   // API messages
   if (error.response && error.response.data.messages !== undefined) {
-    logInfo('API Messages', error.response.data.messages);
+    logInfo('API errors returned to client in messages field', { messages: error.response.data.messages });
     handleApiMessages(error.response.data.messages);
   }
 
   // Single API error
   if (error.response && error.response.data.error_code) {
-    logInfo('API Error', error.response.data.errors);
+    logInfo('API error returned to client as response', {
+      errors: [
+        {
+          error_code: error.response.data.error_code,
+          user_message: error.response.data.user_message,
```

Comment:
Could this break things if there is no `error.response.data.user_message`?  My main concern about this PR is that we don't introduce errors.  We can spend time testing that it does the right thing tomorrow.

Reply:
Actually, I’m not clear on what case this is and why we aren’t using `logAPIErrorResponse(error);` here. You can fill me in tomorrow.

Reply:
Not really, in my opinion.  If user_message isn't there it'll just be undefined.  If _data_ isn't there, we've got bigger problems, but I don't think that's possible.

Reply:
I see regarding user_message. Thanks.
I need to see the code on my computer tomorrow. I’m unclear what this situation is, and the message didn’t clarify.